schema tools.ipynb

```diff
@@ -0,0 +1,367 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
```

Comment:
Let's clear the output.

Let's just put this in a .py script

Reply:
Done.